plugin/src/main/java/io/jenkins/plugins/casc/Configurator.java

```diff
@@ -167,4 +168,26 @@ default CNode describe(T instance, ConfigurationContext context) throws Exceptio
         return mapping;
     }
 
+    /**
+     * Describe Structure of the attributes, as required by the schema.
+     * @param instance
+     * @param context
```

Comment:
Oleg's suggesting that you add the version that this was released in, when you don't know when it will get merged you can add TODO, given we'll be releasing after merging 1.35 seems like less work to me
```suggestion
     * @param context
     * @since 1.35
```